caffe2/onnx/backend.h

```diff
@@ -0,0 +1,169 @@
+#pragma once
+
+#include "caffe2/proto/caffe2.pb.h"
+#include "onnx/onnx_pb.h"
+#include "device.h"
+#include "backend_rep.h"
+
+#include <google/protobuf/text_format.h>
+#include <functional>
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+
+namespace onnx_caffe2 {
+
+struct Caffe2Ops {
+  ::google::protobuf::RepeatedPtrField<caffe2::OperatorDef> ops;
```

Comment:
why not std::vector?

Reply:
Because this structure holds stuff from `RepeatedPtrField` and will flush them to `RepeatedPtrField` in the model. I figured have native type working together maybe better. But it turned out that they don't differ too much...